.github/workflows/tests.yml

```diff
@@ -8,14 +8,14 @@ on:
 
 jobs:
   unit-tests:
-    name: Unit Tests
+    name: Unit Tests & Integration Tests
     runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v1
       - uses: actions/setup-node@v1
         with:
-          node-version: 12.x
+          node-version: 18.x
```

Comment:
why the large node bump to 18? what about 16

Reply:
Because both routing-api and interface node env are on v18 now.